build/admission/gen-admission-secret.sh

```diff
@@ -33,12 +33,12 @@ DNS.3 = ${SERVICE}.${NAMESPACE}.svc
 EOF
 
   openssl genrsa -out ${CERTDIR}/ca.key 2048
-  openssl req -x509 -new -nodes -key ${CERTDIR}/ca.key -subj "/CN=${SERVICE}.${NAMESPACE}.svc" -out ${CERTDIR}/ca.crt
+  openssl req -x509 -new -days 3650 -nodes -key ${CERTDIR}/ca.key -subj "/CN=${SERVICE}.${NAMESPACE}.svc" -out ${CERTDIR}/ca.crt
```

Comment:
it that too large? any reason to set this as 3650?

Reply:
It can be set for one year or ten years, just to avoid the `admisstion` failure caused by the short validity period.